include/ignition/physics/detail/GetEntities.hh

```diff
@@ -214,6 +214,58 @@ namespace ignition
               ->GetWorldOfModel(this->identity));
     }
 
+    /////////////////////////////////////////////////
+    template <typename PolicyT, typename FeaturesT>
+    std::size_t
+    GetNestedModelFromModel::Model<PolicyT, FeaturesT>::GetModelCount() const
+    {
+      return this->template Interface<GetNestedModelFromModel>()
+          ->GetNestedModelCount(this->identity);
+    }
+
+    /////////////////////////////////////////////////
+    template <typename PolicyT, typename FeaturesT>
+    auto GetNestedModelFromModel::Model<PolicyT, FeaturesT>::GetModel(
+        const std::size_t _index) -> ModelPtrType
```

Comment:
I noticed that the declaration of this method doesn't have the `_index` parameter marked as `const` (I noticed the same trend when looking at other features):

https://github.com/ignitionrobotics/ign-physics/blob/ab959226520f908b4996aa52f749ba80d15099b8/include/ignition/physics/GetEntities.hh#L232-L237

Can you help me understand why these don't match, and how this still works/compiles?

Reply:
This has been addressed by Grey in https://github.com/ignitionrobotics/ign-physics/pull/228#discussion_r599292319